docs/user/metacat/source/configuration.rst

```diff
@@ -255,101 +218,111 @@ Deploy Location            The directory where the application is deployed.
+evaluating Metacat by logging into the KNB website, and navigating to "My Profile" -> "Settings" -> "Authentication Token".
 
-Once you have selected a skin, Metacat will open a list of options that apply to the Registry 
-interface. For more information about creating skins, see the Creating a Custom Skin
-section in :doc:`themes`.
+ **Note:** This temporary auth token is only valid for 24 hours, but you can continue manually retrieving short-term tokens
 
-.. figure:: images/screenshots/image023.png
-   :align: center
-   :width: 1100px
+.. _contact DataONE: https://www.dataone.org/contact/
 
-   Configuring Metacat skins.
-   
-Select the checkbox next to your customized skin or and click the 
-``Make <skin_name> default`` radio button. If you do not have a custom skin, 
-select the ``default`` skin. Once you have selected a skin, Metacat will open 
-a list of options that apply to the Registry interface.
+.. _encourage you to join: https://www.dataone.org/jointhenetwork/
+
+
+===============================  =============================================================
+Property                         Description
+===============================  =============================================================
+Token Verification Certificates  Semicolon-separated list of paths to certificate files, each
+                                 containing a single public key (not a certificate chain).
+                                 These will be used to verify incoming request jwt tokens,
+                                 in addition to verifying against the CN server.
+Admin Token Path                 The path to the admin jwt token that will be used in
+                                 dataone-indexer to access the private objects' system metadata.
+===============================  =============================================================
+
+The Admin Token
+~~~~~~~~~~~~~~~
+After obtaining an admin (auth) token, save it to the default path (below). If you wish to save it
+elsewhere, be sure to update the global properties' ``Admin Token Path`` value with your custom path.
+
+ Default Value for the admin (auth) token:
+
+ ::
+
+   /var/metacat/certs/token
+
+Token Verification Certificate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Next, you will need to obtain a DataONE Intermediate Certificate.
+Depending on your needs, there are two available:
+
+ 1. `DataONE Production Intermediate CA Intermediate`_
+
+ 2. `DataONE Test Intermediate CA Certificate`_
+
+ **Note:** the DataONE Intermediate CA certificate is a single certificate, NOT a certificate chain!
 
-Select the lists and modules that you would like to appear in the Registry 
-form-interface by checking the box beside each. When you save the configuration, 
-the customized interface will appear to site visitors.
+.. _DataONE Production Intermediate CA Intermediate: https://raw.githubusercontent.com/DataONEorg/ca/main/DataONETestIntCA/certs/DataONETestIntCA.pem
```

Comment:
this points to the Test int cert, not the prod one

Reply:
Thank you for catching this!